cmd/manager/main.go

```diff
@@ -163,6 +165,11 @@ func Command() *cobra.Command {
 		false, // Set to false for backward compatibility
 		"Restrict cross-namespace resource association through RBAC (eg. referencing Elasticsearch from Kibana)",
 	)
+	cmd.Flags().Bool(
+		operator.EnableLeaderElection,
+		true,
```

Comment:
Do you see any reason for disabling it? (other than: we discovered it's broken/buggy)

Reply:
Discussed through another channel: it's aimed to be an escape hatch in case we reach a deadlock in the leader election process, for some reasons.